Swiftz/ArrayExt.swift

```diff
@@ -206,9 +244,9 @@ extension Array {
 		return res
 	}
 
-	/// Takes a binary function, an initial value, and a list and scans the function across each element
-	/// of a list accumulating the results of successive function calls applied to reduced values from
-	/// the left to the right.
+	/// Takes a binary function, an initial value, and a list and scans the function across each 
```

Comment:
Another thing rewritten!  I'm not sure it's the best way I could have put it, but it's better than this.